install/kubernetes/Makefile

```diff
@@ -40,3 +45,35 @@ docs:
 .PHONY: $(TETRAGON_CHART)/crds-yaml
 $(TETRAGON_CHART)/crds-yaml: $(CRDS)
 	cp -rf $(CRDS)/. $(TETRAGON_CHART)/crds-yaml
+
+# openapi2jsonschema.py script generating JSON schema from the CRD YAML spec.
+.PHONY: openapi2jsonschema.py
+openapi2jsonschema.py:
+	curl -sSfLO https://raw.githubusercontent.com/yannh/kubeconform/$(KUBECONFORM_VERSION)/scripts/$@
+
+# To validate (using openapi2jsonschema.py) default Ruleset policies included in the Helm chart, we need to pass the
+# JSON schema of the TracingPolicy CRD. This target generates such schema. It requires pipenv to be pre-installed.
+.PHONY: generate-jsonschemas
+generate-jsonschemas: $(CRDS)
+	mkdir -p $(JSON_SCHEMAS)/
+	pipenv install
+	FILENAME_FORMAT='{kind}-{fullgroup}' $(PIPENV) run $(PYTHON) openapi2jsonschema.py $(CRDS)/*
+	mv $(REPO_ROOT)/install/kubernetes/*-cilium.io.json $(JSON_SCHEMAS)/
+	pipenv --rm
+
+.PHONY: kubeconform
+kubeconform:
+	@echo "## Testing Helm chart: \"$(TETRAGON_CHART)\""
+	$(HELM) template $(TETRAGON_CHART)-ee . \
+	-f values.yaml \
+	--set crds.installMethod=helm \
+	--set tracingPolicies.default.enabled=true |\
+		kubeconform \
```

Comment:
It seems we don't use `KUBECONFORM_IMAGE` (L6-7) anymore, so we could remove it, but maybe better run kubeconform as a container here? i.e. `docker run --rm -i $(KUBECONFORM_IMAGE)` - then we don't need to install kubeconform in CI extra.

Reply:
Good point. I addressed that in the newest commit. I've tested it locally, but let's wait until GH CI finishes to see if everything works smoothly. If it does, we could ship it, in my opinion.